doc/nrf/libraries/bin/lwm2m_carrier/CHANGELOG.rst

```diff
@@ -18,6 +18,50 @@ For certification status of the released versions, see the following documents,
 * `nRF9161 Mobile network operator certifications`_
 * `nRF9160 Mobile network operator certifications`_
 
+liblwm2m_carrier 3.7.0
+**********************
+
+Release for modem firmware versions 1.3.7, 2.0.2, and 2.0.3.
+
+Size
+====
+
+See :ref:`lwm2m_lib_size` for an explanation of the library size in different scenarios.
+
++-------------------------+---------------+------------+
+|                         | Flash (Bytes) | RAM (Bytes)|
++-------------------------+---------------+------------+
+| Library size            | 78147         | 18828      |
+| (binary)                |               |            |
++-------------------------+---------------+------------+
+| Library size            |               |            |
+| (reference application) |               |            |
++-------------------------+---------------+------------+
+
+Changes
+=======
+
+* Added the new :ref:`lib_downloader` library to the glue layer (:file:`lwm2m_os.c/h`).
+  Removed the deprecated :ref:`lib_download_client` from the glue layer.
+* Added missing documentation for some glue layer functions.
+* Error codes have been improved to more uniquely identify possible errors.
+  In addition, some previously unlisted errors have now been documented.
+  For more information see the following events:
+
+  * :c:macro:`LWM2M_CARRIER_ERROR_INIT`
+  * :c:macro:`LWM2M_CARRIER_ERROR_CONFIGURATION`
+  * :c:macro:`LWM2M_CARRIER_ERROR_FOTA_FAIL`
+  * :c:macro:`LWM2M_CARRIER_ERROR_CONNECT`
+
+* Improved the handling of failures during the download of FOTA images to ensure that the relevant DFU target is reinitialized appropriately before the download is resumed.
+* Optimizations in the CoAP implementation to reduce the memory footprint.
+* Fixed an issue for delayed bootstrap when connecting to 2 servers.
+
+  * Previously, if the second connection failed and caused a delayed boostrap, the first server would disconnect (and connect again once the deferred time has passed).
+  * Now, the first server will remain connected while the LwM2M carrier library is waiting to bootstrap to the second server again.
```

Comment:
>waiting to bootstrap to the second server again.

I'm not sure if this sentence sounds right. Shouldn't it be simply "waiting to bootstrap again."?